rom/dev/tests/test_wdt_activation_and_stoppage.rs

```diff
@@ -0,0 +1,83 @@
+// Licensed under the Apache-2.0 license
+
+use caliptra_builder::{firmware::APP_WITH_UART, firmware::FMC_WITH_UART, ImageOptions};
+use caliptra_common::RomBootStatus::KatStarted;
+use caliptra_hw_model::{DeviceLifecycle, HwModel, SecurityState};
+
+pub mod helpers;
+
+#[test]
+fn test_wdt_activation_and_stoppage() {
+    let security_state = *SecurityState::default()
+        .set_debug_locked(true)
+        .set_device_lifecycle(DeviceLifecycle::Unprovisioned);
+
+    // Build the image we are going to send to ROM to load
+    let image_bundle = caliptra_builder::build_and_sign_image(
+        &FMC_WITH_UART,
+        &APP_WITH_UART,
+        ImageOptions::default(),
+    )
+    .unwrap();
+
+    let rom =
+        caliptra_builder::build_firmware_rom(&caliptra_builder::firmware::ROM_WITH_UART).unwrap();
+    let mut hw = caliptra_hw_model::new(caliptra_hw_model::BootParams {
+        init_params: caliptra_hw_model::InitParams {
+            rom: &rom,
+            security_state,
+            ..Default::default()
+        },
+        ..Default::default()
+    })
+    .unwrap();
+
+    // Ensure we are starting to count from zero.
+    hw.step_until(|m: &mut caliptra_hw_model::ModelEmulated| {
```

Comment:
Don't ever reference the type ModelEmulated. Use the DefaultHwModel alias instead, which uses the correct type depending on the feature flags.

Reply:
Done